internal/routers/routers.go

```diff
@@ -81,8 +85,10 @@ func NewAPIRouter(
 	}
 	verifier := provider.Verifier(config)
 
-	private := r.Group("/")
+	private := r.Group("/api", loggerMiddleware)
 	{
+		private.Use(browserFlow.CorsMiddleware())
```

Comment:
I think you'll have to actually copy/paste the middleware config, you can't just use the one from only one of the agents. Things like trustedOrigins are would be incorrect etc...

Reply:
the origin were setup correctly.. but I didi need to move the CorsMiddleware() the root router.